src/components/Semester/SemesterView.vue

```diff
@@ -342,7 +342,7 @@ export default Vue.extend({
      * Works in conjunction with addCourse()
      * CHANGE WILL ALTER DATA STRUCTURE
      */
-    toFirebaseCourse(course: AppCourse) {
+    toFirebaseCourse(course: AppCourse): FirestoreSemesterCourse {
       return {
         crseId: course.crseId,
         code: `${course.subject} ${course.number}`,
```

Comment:
This function proves that the type declaration of `FirestoreSemesterCourse` is correct. Here we are simply adding a type annotation to show that things type check.

Since this function is unchanged, the stuff we store into the database is also unchanged. Therefore, this migration is safe!